static/app/components/organizations/pageFilters/container.tsx

```diff
@@ -162,6 +163,11 @@ function Container({skipLoadLastUsed, children, ...props}: Props) {
     lastQuery.current = location.query;
   }, [location.query]);
 
+  const [showDesyncedAlert, setShowDesyncedAlert] = useState<boolean>(false);
+  useEffect(() => {
+    setShowDesyncedAlert(desyncedFilters.size > 0);
```

Comment:
Why do you need this? can't you literally just check `desyncedFilters.size > 0` ?

Reply:
Oh because it gets hidden

Reply:
Would it make more sense to have a state that's like `hideDesyncedAlert`. Since right now if you reset a single desynced filter it will run this effect again and even if you've hidden the alert it will pop back up?

Reply:
ahhh didn't catch that, yea i'll switch it over to `hideDesyncedAlert`